Cargo.toml

```diff
@@ -3,3 +3,4 @@ members = ["crates/*"]
 
 [patch.crates-io]
 runix-derive = { path = "crates/runix-derive" }
+runix-types = { path = "crates/runix-types" }
```

Comment:
Why do we need an extra crate for this?


There's also at least flake metadata which would have a typed representation as well.

Reply:
We'll need to use some of these types in flox-types (I actually just copied `Narinfo` from flox-types) and I was thinking it might be better to not pull in all of runix, but I guess it doesn't really matter if we pull in all of runix, and we pull in runix already.